components/chart/index.tsx

```diff
@@ -20,6 +21,9 @@ function Chart({
   const rootContent = data[0];
   const [zoomLevel, setZoomLevel] = useState(100);
   const [selectedModules, setSelectedModules] = useState([]);
+  const [Description, setDescription] = useState([]);
```

Comment:
let's follow the default naming conventions
```suggestion
  const [description, setDescription] = useState([]);
```
Also, I think the description should be a string.

Reply:
done